rdf_differ/config.py

```diff
@@ -9,9 +9,8 @@
 
 def get_envs() -> dict:
     return {
-        'basedir': os.environ.get('BASEDIR', './basedir'),
         'filename': os.environ.get('FILENAME', 'file'),
-        'endpoint': os.environ['ENDPOINT']
+        'endpoint': os.environ.get('ENDPOINT', 'http://localhost:3030')
```

Comment:
this is a strong assumption, but fine, let it be; we need to make sure that the env variable is there on deployment.

Reply:
I forgot to remove it from here. 
Probably a good update to the project would be to add environment variables in a make command.